### ISSUE TITLE
Inconsistency between MYSQL and PLpgSQL

### ISSUE THREAD

**Issue by mmkal** [modeldba/sql-surveyor#2]
Related to #1 but thought I'd make a separate issue:

PLpgSQL treats aliases differently from MYSQL (and the MYSQL way seems more correct - if this is expected behaviour, I couldn't find it in the docs):

```js
let sqlSurveyor = require("sql-surveyor")

function getAliasF({ dialect, alias }) {
  let surveyor = new sqlSurveyor.SQLSurveyor(dialect)
  let result = surveyor.survey('select f.id from foo as f')
  return result.getTableFromAlias(alias)
}

console.log(getAliasF({ dialect: sqlSurveyor.SQLDialect.MYSQL, alias: 'f' }) // prints 'foo', as expected
console.log(getAliasF({ dialect: sqlSurveyor.SQLDialect.PLpgSQL, alias: 'f' }) // prints null

console.log(getAliasF({ dialect: sqlSurveyor.SQLDialect.MYSQL, alias: 'as f' }) // prints null
console.log(getAliasF({ dialect: sqlSurveyor.SQLDialect.PLpgSQL, alias: 'as f' }) // prints 'foo', as expected
```

Another difference (noted in #1):

```js
// result.outputColumns[0] with MYSQL:
    OutputColumn {
      columnName: 'f.id',
      columnAlias: null,
      tableName: 'foo',
      tableAlias: 'f'
    }

// result.outputColumns[0] with PLpgSQL:
    OutputColumn {
      columnName: 'f.id',
      columnAlias: null,
      tableName: 'f',
      tableAlias: null
    }
```

**Comment by mtriff** [comment]
Thanks for the bug report! The output should be identical between the two dialects. The MySQL output is the expected behaviour.

I'll have a fix for this available shortly.

**Comment by mtriff** [comment]
This has been fixed in [1.2.2](https://github.com/modeldba/sql-surveyor/releases/tag/1.2.2).

Both MySQL and PL/pgSQL now output:
```
outputColumns: [
      OutputColumn {
        columnName: 'f.id',
        columnAlias: null,
        tableName: 'foo',
        tableAlias: 'f'
      }
    ],
```

Thanks again for reporting!

**Comment by mmkal** [comment]
Wow that was fast! I'll try it out!
